src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParametersStateTest.kt

```diff
@@ -0,0 +1,141 @@
+        Arguments.of(
+            { state.stoppingCondition = StoppingCondition.MAXFITNESSEVALUATIONS },
+            mutableListOf("-Dstopping_condition=MAXFITNESSEVALUATIONS")
+        ),
+        Arguments.of(
+            { state.searchBudget = 42 },
+            mutableListOf("-Dsearch_budget=42")
+        ),
+        Arguments.of(
+            { state.initializationTimeout = 69 },
+            mutableListOf("-Dinitialization_timeout=69")
+        ),
+        Arguments.of(
+            { state.minimizationTimeout = 256 },
+            mutableListOf("-Dminimization_timeout=256")
+        ),
+        Arguments.of(
+            { state.assertionTimeout = 61 },
+            mutableListOf("-Dassertion_timeout=61"),
+        ),
+
+        Arguments.of(
+            { state.junitCheckTimeout = 59 },
+            mutableListOf("-Djunit_check_timeout=59")
+        ),
+        Arguments.of(
+            { state.populationLimit = PopulationLimit.STATEMENTS },
+            mutableListOf("-Dpopulation_limit=STATEMENTS")
+        ),
+        Arguments.of(
+            { state.population = 128 },
+            mutableListOf("-Dpopulation=128")
+        )
+    )
+
+    @Test
+    fun testDefaultStateNotAffected() {
+        state.stoppingCondition = StoppingCondition.TIMEDELTA
+        state.searchBudget = 123
+        state.initializationTimeout = 890
+        state.minimizationTimeout = 534
+        state.assertionTimeout = 231
+        state.junitCheckTimeout = 217
+        state.populationLimit = PopulationLimit.TESTS
+        state.population = 1
+
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.stoppingCondition.name)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("stoppingCondition"))
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.searchBudget)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("searchBudget").toInt())
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.initializationTimeout)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("initializationTimeout").toInt())
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.minimizationTimeout)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("minimizationTimeout").toInt())
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.assertionTimeout)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("assertionTimeout").toInt())
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.junitCheckTimeout)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("junitCheckTimeout").toInt())
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.populationLimit.name)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("populationLimit"))
+        Assertions.assertThat(QuickAccessParametersState.DefaultState.population)
+            .isEqualTo(TestGenieDefaultsBundle.defaultValue("population").toInt())
```

Comment:
Why don't you import the bundle and the QuickAccessParameters:
`val state = QuickAccessParametersState.DefaultState`

Reply:
Good point! I don't know why i haven't come up with that myself

Reply:
Done